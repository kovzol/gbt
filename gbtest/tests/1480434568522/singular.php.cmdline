#!/bin/sh
cd tests/1480434568522
cat singular.php.in | /usr/bin/timeout -k 120 60 /usr/bin/time /usr/bin/Singular > singular.php.out 2>singular.php.err