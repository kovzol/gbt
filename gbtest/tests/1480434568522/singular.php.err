0.00user 0.00system 0:00.00elapsed 88%CPU (0avgtext+0avgdata 9904maxresident)k
0inputs+8outputs (0major+405minor)pagefaults 0swaps
