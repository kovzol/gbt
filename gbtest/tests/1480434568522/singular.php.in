ring r=0,(v20,v21,v19,v16,v15,v12,v11,v10,v9,v8,v7,v6,v5,v4,v3,v1,v2),dp;ideal I=-1*v11,-1*v9,-1+v12,v10,-1+v7,v6,v8,v5,-1*v11*v10+v12*v9+v11*v4-1*v9*v4-1*v12*v3+v10*v3,-1*v8+v6-1*v7*v6+v8*v5,-1*v7*v6+v8*v5+v7*v4-1*v5*v4-1*v8*v3+v6*v3,-1+v19^2-1*v2^2+2*v1-1*v1^2,v21^4-2*v21^2*v20^2+v20^4-2*v21^2*v19^2-2*v20^2*v19^2+v19^4,-1*v16^2+2*v15-1*v15^2+v4^2-2*v3+v3^2,v20^2-1*v4^2-1*v3^2+2*v4*v2-1*v2^2+2*v3*v1-1*v1^2,1-1*v16^2-1*v15^2+2*v16*v4-2*v3+2*v15*v3,v21^2-1*v16^2-1*v15^2+2*v16*v2-1*v2^2+2*v15*v1-1*v1^2; option(redSB); groebner(I);