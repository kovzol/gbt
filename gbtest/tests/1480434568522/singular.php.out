                     SINGULAR                                 /
 A Computer Algebra System for Polynomial Computations       /   version 4.0.3
                                                           0<
 by: W. Decker, G.-M. Greuel, G. Pfister, H. Schoenemann     \   Jan 2016
FB Mathematik der Universitaet, D-67653 Kaiserslautern        \  Debian 4.0.3+ds-1
_[1]=v3
_[2]=v4
_[3]=v5
_[4]=v6
_[5]=v7-1
_[6]=v8
_[7]=v9
_[8]=v10
_[9]=v11
_[10]=v12-1
_[11]=2*v15-1
_[12]=4*v16^2-3
_[13]=v19^2-v1^2-v2^2+2*v1-1
_[14]=v21^2-v1^2+2*v16*v2-v2^2+v1-1
_[15]=v20^2-v1^2-v2^2
_[16]=3*v1^4-4*v16*v1^2*v2+6*v1^2*v2^2-4*v16*v2^3+3*v2^4-6*v1^3+4*v16*v1*v2-6*v1*v2^2+3*v1^2+v2^2
> Auf Wiedersehen.
